plugins/chat/assets/javascripts/discourse/components/chat-channel-unread-indicator.gjs

```diff
@@ -11,6 +11,7 @@ export default class ChatChannelUnreadIndicator extends Component {
   get showUnreadIndicator() {
     return (
       this.args.channel.tracking.unreadCount > 0 ||
+      this.args.channel.tracking.mentionCount > 0 ||
```

Comment:
For mentions within threads (ie. urgent), we don't want the unread indicator to disappear when we open the channel since user hasn't opened the thread where they were mentioned.

Previously we relied on `unreadThreadsCountSinceLastViewed` which only shows thread unreads since the channel was last opened, which works as expected for low urgency thread replies.